src/mono/mono/mini/interp/transform.h

```diff
@@ -447,7 +447,11 @@ int
 interp_create_local (TransformData *td, MonoType *type);
 
 void
-interp_foreach_local_var (TransformData *td, InterpInst *ins, gpointer data, void (*callback)(TransformData*, int, gpointer));
+interp_foreach_ins_var (TransformData *td, InterpInst *ins, gpointer data, void (*callback)(TransformData*, int*, gpointer));
+
+void
+interp_foreach_ins_svar (TransformData *td, InterpInst *ins, gpointer data, void (*callback)(TransformData*, int*, gpointer));
```

Comment:
The functions could be easily mismatched due to similar names and the same args.